kernel/packages/unity-interface/BrowserInterface.ts

```diff
@@ -204,6 +205,18 @@ export class BrowserInterface {
         }
         break
       }
+      case 'StartStateMode': {
```

Comment:
I don't like this name so much. Maybe something like `StartStatefulMode`? What do you think?

Reply:
Maybe something with the `sync` word in it? `StartSceneSync` `StartSceneSyncMode`?